core/src/main/scala/sttp/tapir/util/ShadowedEndpointChecker.scala

```diff
@@ -0,0 +1,32 @@
+package sttp.tapir.util
+
+import sttp.tapir.{Endpoint, ShadowedEndpoint}
+
+object ShadowedEndpointChecker {
+
+  def findShadowedEndpoints(endpoints: List[Endpoint[_, _, _, _]]): List[ShadowedEndpoint] = {
```

Comment:
Maybe the object could be named e.g. `FindShadowedEndpoints` and the method `apply`, then usage would be cleaner, without repeating twice the same? That is: `FindShadowedEndpoints(endpoints)`